agent/engine/docker_image_manager_test.go

```diff
@@ -1084,7 +1084,7 @@ func TestDeleteImageNotFoundError(t *testing.T) {
 	}
 	imageState, _ := imageManager.getImageState(imageInspected.ID)
 	client.EXPECT().RemoveImage(gomock.Any(), container.Image, dockerclient.RemoveImageTimeout).Return(
-		errors.New("no such image"))
+		errors.New("no such image: " + container.Image))
```

Comment:
Can you test the old behavior as well?  We want to ensure that we don't have a regression against older versions of Docker that may return "no such image" instead of including the suffix.